elastic/kibana
# [Uptime] [Bug] Remove some default common DatePicker values

## Summary

Resolves #37299.

There are several default common options provided by `EuiSuperDatePicker` that we don't need for our solution's purposes at the moment. Because we utilize histogram aggs that require an interval value, we can't easily accept a `from` and `to` date range with a difference of `0ms`.

This patch defines a default list of values to supply for common choices, and supplies it to our date picker.

## Testing
To ensure this fix is working:

1. Start up Heartbeat
2. Open the Uptime UI
3. Open the date picker's dropdown
4. Select each of the "common" values and ensure the app handles them appropriately
![image](https://user-images.githubusercontent.com/18429259/58587794-40de6780-822c-11e9-9228-42e1db91424f.png)



### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
